private/PROPOSAL/medium/Medium.cxx

```diff
@@ -710,6 +710,33 @@ AntaresWater::AntaresWater(double rho)
 
 }
 
+// chemical composition is normalized with respect to hydrogen: calculate a
+// total atomic weight that yields two hydrogen atoms; use this weight to
+// calculate the numbers of atoms for all other elements
+CascadiaBasinWater::CascadiaBasinWater(double rho)
+    : Medium(
+        "cascadiabasinwater",
+        rho,
+        75.,       // I
+        -3.5017,   // C
+        0.09116,   // a
+        3.4773,    // m
+        0.2400,    // X0
+        2.8004,    // X1
+        0.,        // d0
+        1.0400322, // massDensity
+        {
+            std::shared_ptr<Components::Component>(new Components::Hydrogen(2.0)),
```

Comment:
Maybe make_shared would be appropriate here?

https://en.cppreference.com/w/cpp/memory/shared_ptr/make_shared

Reply:
Yes, makes sense. I've to admit that I simply copied the Antares water and then implemented my changes.

Reply:
Can I simply implement the proposed changes, commit to my fork, and the pull request gets automatically updated?

Reply:
Yes